libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -1081,3 +1087,40 @@ def upsert(
                 "Failed to upsert entities: %s error: %s", self.collection_name, exc
             )
             raise exc
+
+    def get_documents_by_ids(self, ids: int | str | List[int | str]) -> List[Document]:
+        # Generating filtering expr for passing to query function
```

Comment:
nit: map into the same code path, or force users to always think in terms of batch (it's good bias since the code involves round trips between client code and server)

if not isinstance(ids, (list, tuple)):
   ids = [ids]